config/initializers/time_formats.rb

```diff
@@ -0,0 +1 @@
+Time::DATE_FORMATS[:ddmmyy] = '%d/%m/%Y'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.